VmAgent.Core/Interfaces/ProcessRunner.cs

```diff
@@ -18,6 +19,11 @@ public class ProcessRunner : BaseSessionHostRunner
     {
         private readonly IProcessWrapper _processWrapper;
 
+        /// <summary>
+        /// The name of the file where the console logs for the server are captured.
+        /// </summary>
+        private const string ConsoleLogCaptureFileName = "PF_ConsoleLogs.txt";
```

Comment:
maybe we can move this to BaseSessionHostRunner because now this file name is defined in both ProcessRunner.cs and DockerContainerEngine.cs?